6.bulk_Morphology_Elastic_Network/2.test_model/notebooks/1.test_regression_multi_output.ipynb

```diff
@@ -1,428 +1,512 @@
 {
```

Comment:
<p>It looks like there was an error here, just wanted to bring it to your attention</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Error fixed, New data was passed in</p>